app/javascript/comments.vue

```diff
@@ -294,6 +294,16 @@ export default {
       }
     },
     postComment() {
+      if (this.commentableType === 'Report' && this.isRole('mentor')) {
+        const notChecked = !this.checkId
+        if (notChecked) {
+          if (
+            !window.confirm('日報を確認済みにしていませんがよろしいですか？')
+          ) {
+            return
+          }
+        }
```

Comment:
不必要なブロックの入れ子は避けるべきなので下記はいかがでしょうか？？

```suggestion
        if (notChecked && !window.confirm('日報を確認済みにしていませんがよろしいですか？')) {
          return
        }
```

Reply:
@komagata 
コメントありがとうございます。
ご提案頂いたコードに修正しました。
不必要なブロックの入れ子にならないように自分でも気づけるように注意したいと思います。
ありがとうございます！